.github/workflows/pr.yml

```diff
@@ -0,0 +1,25 @@
+name: pr
+on: pull_request
+jobs:
+  lint:
```

Comment:
just fyi, i'm starting to see people set permissions on these. idk what kind of permissions these particular actions require but it's worth considering

```
permissions:
  contents: read
```

Reply:
Will look at docs.

Reply:
Added.